app/settings.py

```diff
@@ -5,7 +5,8 @@
 from requests.adapters import HTTPAdapter
 
 
-LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
+LOGGING_LEVEL = logging.getLevelName(os.getenv("LOGGING_LEVEL", "DEBUG"))
+LOGGING_FORMAT = "%(asctime)s.%(msecs)06dZ|%(levelname)s: sdx-collect: %(message)s"
```

Comment:
Does this include the T in between the yyyy-m-ddThh:MM:ss.mmmmmmZ? (with the M being minutes and m being milliseconds)